coops-ui/src/components/AnnualReport/BCorp/ARDate.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <v-card flat class="container">
+    <div class="timeline">
+        <label>Annual Report Date</label>
+        <span class="date ar-date">{{this.toReadableDate(this.nextARDate)}}</span>
+        <label>Filing Date</label>
+        <span class="date file-date">Today ({{this.toReadableDate(new Date())}})</span>
+      </div>
+  </v-card>
+</template>w
+<script lang="ts">
+import { Component } from 'vue-property-decorator'
+import { mapState } from 'vuex'
+import DateMixin from '@/mixins/date-mixin'
+
+@Component({
+  computed: {
+    ...mapState(['nextARDate'])
+  }
+})
+export default class ARDate extends DateMixin {
+    readonly nextARDate: string
+}
+</script>
+<style lang="stylus" scoped>
+  @import '../../../assets/styles/theme.styl'
+
+  .container
+    background: #FFFFFF
+
+    .timeline
+      display: flex
+      justify-content space-between
+
+      label
+        font-weight: 500
+
+      .date
+        font-family: "BC Sans"
```

Comment:
This is incorrect because there is no fallback font stack.
Please discuss this with @jeznorth on the reasoning for this font here. This should probably be controlled by the application instead of in this (sub)component.

Reply:
K sounds good, i will do that. It was a request of our designer, I will research further. Thanks!

Reply:
The font declaration has been removed in the latest commit. It was decided that this requirement would be handled application-wide in https://app.zenhub.com/workspaces/entity-5bf2f2164b5806bc2bf60531/issues/bcgov/entity/1369